tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/section_participation_concept_suppression_test.py

```diff
@@ -0,0 +1,116 @@
+"""
+Integration test for section_participation_concept_suppression module
+
+Record suppress (drop the rows) responses related to the section_particiption concept (*_concept_id = 903632) in both registered and controlled tiers.
+
+
+Original Issue: DC-1641
+"""
+
+# Python Imports
+import os
+
+# Project imports
+from app_identity import PROJECT_ID
+from cdr_cleaner.cleaning_rules.section_participation_concept_suppression import SectionParticipationConceptSuppression
+from tests.integration_tests.data_steward.cdr_cleaner.cleaning_rules.bigquery_tests_base import BaseTest
+from common import OBSERVATION
+
+# Third party imports
+from dateutil import parser
+
+
+class SectionParticipationConceptSuppressionTest(BaseTest.CleaningRulesTestBase
+                                                ):
+
+    @classmethod
+    def setUpClass(cls):
+        print('**************************************************************')
+        print(cls.__name__)
+        print('**************************************************************')
+
+        super().initialize_class_vars()
+
+        # set the test project identifier
+        project_id = os.environ.get(PROJECT_ID)
+        cls.project_id = project_id
+
+        # set the expected test datasets
+        dataset_id = os.environ.get('COMBINED_DATASET_ID')
+        cls.dataset_id = dataset_id
+        sandbox_id = dataset_id + '_sandbox'
+        cls.sandbox_id = sandbox_id
+        cls.vocabulary_id = os.environ.get('VOCABULARY_DATASET')
+
+        cls.rule_instance = SectionParticipationConceptSuppression(
+            project_id, dataset_id, sandbox_id)
+
+        cls.fq_table_names = [
+            f'{project_id}.{dataset_id}.{OBSERVATION}',
+        ]
+
+        cls.fq_sandbox_table_names = [
+            f'{project_id}.{sandbox_id}.{cls.rule_instance.sandbox_table_for(OBSERVATION)}',
+        ]
+
+        cls.vocab_tables = ['concept']
```

Comment:
You might need to add `cls.vocab_tables` to `cls.fq_table_names` so those tables are cleaned after the test. Currently I believe the base cleaning test cleans `cls.fq_table_names` and `cls.fq_sandbox_table_names`.

Reply:
Will do